tests/src/Functional/DefaultCommentTypeTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\Tests\lightning\Functional;
+
+use Drupal\comment\Entity\CommentType;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * @group lightning
+ */
+class DefaultCommentTypeTest extends BrowserTestBase {
```

Comment:
Thank you for writing this test, but I think we can actually remove it. I say that for two reasons: first, it installs Lightning, which will make it incredibly slow; and second, it's really just testing the optional config system in core, which is already very well-covered by core's tests. So there's no need for it here. :)